### PR TITLE
[DOCFIX] Update Contributor Tools

### PR DESCRIPTION
**Pull request by ZacBlanco**
- removed thriftGen, loadUfs from list of commands
- lowercased filename of developer-tips yaml file
- Updated wording and sections

### CODE REVIEW

File: docs/_data/table/en/developer-tips.yml [hunk 1]
```diff
@@ -45,9 +45,6 @@ copyDir:
   Copy the <code>PATH</code> to all worker nodes.
 clearCache:
   Clear OS buffer cache of the machine. This command needs the root permission.
-thriftGen:
-  Generate all thrift code. See <a href="#modifying-a-thrift-rpc-definition">Change a Thrift RPC
-  definition</a>.
 protoGen:
```
Comment (apc999) [inline docs/_data/table/en/developer-tips.yml]: this is gone too, right?

File: docs/en/contributor/contributor-tools.md [hunk 2]
```diff
@@ -154,8 +169,9 @@ $ mvn test -pl underfs/hdfs -PufsContractTest -DtestHdfsBaseDir=hdfs://ip:port/a
 leverage the Scala shell, as discussed in this
 [blog](http://scala4fun.tumblr.com/post/84791653967/interactivejavacoding).
 
-- The fuse tests are ignored if the libfuse library is missing. To run those tests, please install the correct libraries
-mentioned in [this page]({{ '/en/api/POSIX-API.html' | relativize_url }}#requirements).
+- The fuse tests are ignored if the `libfuse` library is missing.
+To run those tests, please install the correct libraries mentioned in
+[the Alluxio FUSE documentation]({{ '/en/api/FUSE-API.html' | relativize_url }}#requirements).
```
Comment (apc999) [inline docs/en/contributor/contributor-tools.md]: I think this has been renamed to Alluxio POSIX API , also with filename to be `/en/api/POSIX-API.html` . please double check

File: docs/_data/table/en/developer-tips.yml (hunk 1 above)
Comment (ZacBlanco) [reply docs/_data/table/en/developer-tips.yml]: fixed

File: docs/en/contributor/contributor-tools.md (hunk 2 above)
Comment (ZacBlanco) [reply docs/en/contributor/contributor-tools.md]: Good catch. This went in before I could pull master with the updates. I'll fix it

**Pull request merged**